common/src/main/java/com/wynntils/core/features/overlays/OverlayPosition.java

```diff
@@ -6,10 +6,12 @@
 
 import com.wynntils.mc.render.HorizontalAlignment;
 import com.wynntils.mc.render.VerticalAlignment;
+import java.io.Serializable;
+import java.util.Objects;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class OverlayPosition {
+public class OverlayPosition implements Serializable {
```

Comment:
Is this necessary? Serialization is soo flawed and very tricky to get right. Is it json stuff that requires this?

Reply:
Yeah it just needed the implementation, no other changes.